packages/reports/addon/components/filter-values/dimension-select.ts

```diff
@@ -59,16 +76,18 @@ export default class DimensionSelectComponent extends Component<DimensionSelectC
     if (this.searchTerm !== undefined) {
       return undefined; // we are searching, so only show search results
     }
-
     return this.dimensionValues;
   }
 
-  get selectedDimensions(): NaviDimensionModel[] {
+  get selectedDimensions(): ExtraNaviDimModel[] {
     const { dimensionColumn } = this;
     const { values } = this.args.filter;
     if (values !== undefined) {
       const dimensionModelFactory = getOwner(this).factoryFor('model:navi-dimension');
-      return values.map((value) => dimensionModelFactory.create({ value, dimensionColumn }));
+      return values.map(
+        (value) =>
+          new ExtraNaviDimModel(dimensionModelFactory.create({ value, dimensionColumn }), { manualInputEntry: false })
```

Comment:
We can't know for sure whether these were selected from the dropdown or manually entered unless we look them up
```suggestion
          new ExtraNaviDimModel(dimensionModelFactory.create({ value, dimensionColumn }), { manualInputEntry: undefined })
```